js/ElectrodeBridge.js

```diff
@@ -321,4 +321,4 @@ export const DispatchMode = {
 
 const electrodeBridge = new ElectrodeBridge()
 electrodeBridge.setMaxListeners(MAX_LISTENERS)
-export electrodeBridge
+export { electrodeBridge }
```

Comment:
If anyone is directly using the electrodeBridge, wouldn't the import statement require it to be wrapped into {electrodeBridge}?

Reply:
Yes, this is already the case, prior to this PR.
This code is ending up as being exactly the same as it was before my fix for listeners:
```
https://github.com/electrode-io/react-native-electrode-bridge/blob/b1a56505da99ce557ec45de3fb68873dfa124bc7/js/ElectrodeBridge.js#L321
```
It's a named export, so you have to explicitly for `import { electrodeBridge }` this is what is currently done in APIs, so it's not changing anything.

Reply:
`export electrodeBridge` is not a valid ES6 code statement. It doesn't 'build' at all.